src/js/components/Button/StyledButtonKind.js

```diff
@@ -204,12 +204,19 @@ const fillStyle = (fillContainer) => {
   return undefined;
 };
 
-const plainStyle = css`
+const plainStyle = (props) => css`
   outline: none;
   border: none;
   padding: 0;
   text-align: inherit;
   color: inherit;
+  ${props.icon &&
```

Comment:
hmm... this feels like the opposite of "plain", where we tend to turn off styling. What if the caller wants to create a custom kind of badge that is an SVG styled to be in the upper right corner?

Reply:
I'm not following this use case so much -- previously we had svg styling in `plainStyle` that mimicked what happened in `basicStyle` which we lost in the recent changes. I would expect this change to restore the styling we had previously supported

Reply:
Ah, interesting. OK.

Reply:
We had it implemented differently (using vertical-align bottom as opposed to display flex) but that styling did exist in both `plainStyle` and `basicStyle` -- we can discuss more later in the week if you're not feeling like this is the right approach